trading_calendars/common_holidays.py

```diff
@@ -60,6 +75,62 @@ def whit_monday(start_date=None, end_date=None):
     )
 
 
+def corpus_christi(start_date=None, end_date=None):
```

Comment:
Looks like we also have these defined in some of the others calendars, should we update those to use these new ones?

Reply:
Ya, I was planning on updating this after those merged actually.

Reply:
Cool that's what I figured (thought it would be good to have a comment here so we don't forget)

Reply:
Yup thanks 👍

Reply:
@yankees714 I made these updates, mind double checking them for me?